feature-changeaddress/src/main/kotlin/com/feature/changeaddress/ChangeAddressViewModel.kt

```diff
@@ -127,6 +127,10 @@ internal class ChangeAddressViewModel(
     }
   }
 
+  fun onErrorDialogDismissed() {
+    _uiState.update { it.copy(errorMessage = null) }
+  }
```

Comment:
So this is going to be for a wide error for the entire screen, which will be there along with the errors per-field which can also have an error right?

Reply:
Correct!